### PR TITLE
fix: overflow pixels on PageHeader Widget

### PR DESCRIPTION
**Pull request by Mawulijo**
**Please follow this checklist. Please check each appropriate box (put an 'x' or check it after creating the PR).**
- [ ] REQUIRED: Do you have an Issue **assigned to you** within the v1 milestone for this PR?  Put the Issue number here:
- [x] Provided detailed pull request description and a succinct title (consider template below for guidance).
- [x] Tested your changes, especially after any code review iterations.
- [x] Included any relevant screenshots of UI updates.
- [x] Followed the [Contributor Guidelines](https://github.com/WorldHealthOrganization/app/blob/master/docs/CONTRIBUTING.md).
- [ ] Verified all contributions are properly licensed pursuant to the [LICENSE](https://github.com/WorldHealthOrganization/app/blob/master/LICENSE) file in the root of the repository.
- [x] Verified your name is in the [content/credits.yaml](https://github.com/WorldHealthOrganization/app/blob/master/content/credits.yaml) file (if you want it to be).

After all boxes above are checked, request and receive an Approved review from any team member knowledgable in the area (TODO team member list).  Once approved, the team member will assign your review to a Committer or use the `needs-merge` label.

## What does this PR accomplish?
Fixes overflow pixels on PageHeader

This issues is seen on pages with longer text on the page header. WHO Myth-busters in this case.

## Did you add any dependencies?
No

## How did you test the change?
Tested it Iphone 11 Pro Max (IOS Simulator)

Screenshot 2020-03-31 at 7 57 26 AM

### CODE REVIEW

File: client/flutter/ios/Podfile.lock [hunk 1]
```diff
@@ -162,4 +162,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: 083258d7f5e80b42ea9bfee905fe93049bc04c64
 
-COCOAPODS: 1.9.1
+COCOAPODS: 1.8.4
```
Comment (epicfaace) [inline client/flutter/ios/Podfile.lock]: Can you remove this change?

File: client/flutter/pubspec.lock [hunk 2]
```diff
@@ -319,7 +319,7 @@ packages:
       name: test_api
       url: "https://pub.dartlang.org"
     source: hosted
-    version: "0.2.11"
+    version: "0.2.15"
```
Comment (epicfaace) [inline client/flutter/pubspec.lock]: Can you remove these changes?

File: client/flutter/ios/Podfile.lock (hunk 1 above)
Comment (Mawulijo) [reply client/flutter/ios/Podfile.lock]: Sure, will do

File: client/flutter/pubspec.lock (hunk 2 above)
Comment (Mawulijo) [reply client/flutter/pubspec.lock]: sure, reverted

**Comment by patniemeyer** [comment]
This changes the layout significantly and I don't know that it addresses the overflow issue...  Thank you for your effort but let's close this for now.

**Pull request closed without merging**